boards/stm32f3discovery/doc.txt

```diff
@@ -15,7 +15,8 @@ and 3-axis magnetometer).
 The board does however not provide any radio capabilities, radio devices have
 to be connected externally via I2C, SPI, UART or similar.
 
-See [this page](https://github.com/RIOT-OS/RIOT/wiki/Getting-started-with-STM32F%5B0%7C3%7C4%5Ddiscovery-boards)
+See [this page]
+(https://www.st.com/bin/ecommerce/api/image.PF254044.en.feature-description-include-personalized-no-cpn-large.jpg)
```

Comment:
This just points to an image, not the getting started guide. BTW the link is still valid.

Reply:
OK, then we should revert it. It was obviously a false positive of the tool.

Reply:
Fixed.